.husky/pre-commit

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+cd "$(dirname -- "$0")/../."
+yarn build
+
+git add .
+
+npx lint-staged
```

Comment:
IIRC husky runs these scripts (`./.husky/*`) with `node_modules/.bin` in its pseudo-$PATH, so `npx` is not needed here.

```suggestion
lint-staged
```